static/src/javascripts/projects/journalism/snippet-feedback.js

```diff
@@ -3,16 +3,27 @@ import fastdom from 'lib/fastdom-promise';
 import mediator from 'lib/mediator';
 import ophan from 'ophan/ng';
 import { getViewport } from 'lib/detect';
+import { SnippetFourVariants } from 'common/modules/experiments/tests/snippet-a-a1-b-b1';
+import { getAssignedVariant } from 'common/modules/experiments/utils';
 
 const SnippetFeedback = (options: { scroll: boolean } = { scroll: true }) => {
     let snippets = [...document.querySelectorAll('.explainer-snippet--new')];
+    const variant = getAssignedVariant(SnippetFourVariants);
 
     snippets.forEach(snippet => {
         const { snippetId, snippetType } = snippet.dataset;
         if (!snippetId || !snippetType) {
             return;
         }
-        const component = `snippet_${snippetType}`;
+        const component = `snippet_${snippetType}${variant
+            ? `_${variant.id}`
+            : ''}`;
+
+        if (variant) {
+            variant.id.split('').forEach(c => {
```

Comment:
why do we need to split here, isn't `variant.id` just e.g. `a1`?

Reply:
I use `a`/`b` to distinguish between light/dark background, and `0`/`1` between plus/minus and arrow down/arrow up icons :-)

Reply:
👍 i see what's going on now!